pkg/kube/configmap/configmap_test.go

```diff
@@ -70,6 +70,21 @@ func TestReadData(t *testing.T) {
 	assert.Equal(t, "value2", data["key2"])
 }
 
+func TestReadFileLikeField(t *testing.T) {
```

Comment:
can we add a test case that will check for bad input to make sure we handle that correctly?